<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRadhard2Flashtool</class>
 <widget class="QDialog" name="QFRadhard2Flashtool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radhard2 Flash Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="cam_radhard2.qrc">
    <normaloff>:/cam_radhard2_logo.png</normaloff>:/cam_radhard2_logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> Settings </string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;auto. flash on connect:</string>
        </property>
        <property name="buddy">
         <cstring>chkAutoload</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="chkAutoload">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>auto. flash b&amp;itfile:</string>
        </property>
        <property name="buddy">
         <cstring>edtAutoBitfile</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="edtAutoBitfile"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnLoad1">
          <property name="toolTip">
           <string>select a bitfile from the filesystem ...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="cam_radhard2.qrc">
            <normaloff>:/cam_radhard2/fileopen.png</normaloff>:/cam_radhard2/fileopen.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&amp;Retries:</string>
        </property>
        <property name="buddy">
         <cstring>spinRetries</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinRetries">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string> Flash Tool </string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Bitfile:</string>
        </property>
        <property name="buddy">
         <cstring>edtFile</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="edtFile"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnLoad">
          <property name="toolTip">
           <string>select a bitfile from the filesystem ...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="cam_radhard2.qrc">
            <normaloff>:/cam_radhard2/fileopen.png</normaloff>:/cam_radhard2/fileopen.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnFlash">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Flash</string>
          </property>
          <property name="icon">
           <iconset resource="cam_radhard2.qrc">
            <normaloff>:/cam_radhard2/flash.png</normaloff>:/cam_radhard2/flash.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Flash Output:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="edtSuccess">
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset resource="cam_radhard2.qrc">
         <normaloff>:/cam_radhard2/dialog_close.png</normaloff>:/cam_radhard2/dialog_close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkAutoload</tabstop>
  <tabstop>edtAutoBitfile</tabstop>
  <tabstop>btnLoad1</tabstop>
  <tabstop>spinRetries</tabstop>
  <tabstop>edtFile</tabstop>
  <tabstop>btnLoad</tabstop>
  <tabstop>btnFlash</tabstop>
  <tabstop>edtSuccess</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="cam_radhard2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QFRadhard2Flashtool</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
